### DISCUSSION TITLE
Subscribe button is gone

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#8656] [Mobile] [answered]
![Screenshot](https://user-images.githubusercontent.com/39104740/145561280-f900b24b-ece4-4bbd-9914-062973b4589b.jpg)

**Comment by brianlovin** [comment, marked as answer]
Hey @nt4f04uNd — thanks for the note. We've moved this control under the `···` menu next to the issue title. In the next version of the app, this control will move to the top-level title bar on this view, making it accessible from any scroll position.
